content/_changelogs/10.9.0.md

```diff
@@ -0,0 +1,43 @@
+## 10.9.0
+
+_Released 9/27/2022_
+
+**Features:**
+
+- Added support for requiring dependencies within the
+  [`cy.origin()`](/api/commands/origin) callback. See the
+  [`Cypress.require()`](/api/cypress-api/require) and
+  [`cy.origin()`](/api/commands/origin#Dependencies-Sharing-Code) docs for more
+  information.
+- Added support for re-using session data cached by
+  [`cy.session()`](/api/commands/session) across specs via the
+  `cacheAcrossSpecs` option. Addresses
+  [#17710](https://github.com/cypress-io/cypress/issues/17710).
+- Added support for advanced dev server configuration via an async function that
+  can optionally modify the dev server config. Addresses
+  [#23302](https://github.com/cypress-io/cypress/issues/23302).
+
+**Bugfixes:**
```

Comment:
Missing bugfix [#23409](https://github.com/cypress-io/cypress/issues/23409), and we're expecting to get feature [#21720](https://github.com/cypress-io/cypress/issues/21720) in today

Reply:
Added #23409 in https://github.com/cypress-io/cypress-documentation/pull/4752/commits/815d89147c62bcb37751467060979be289701e15. I'll make sure #21720 is added once it's merged.

Reply:
Added #21720: https://github.com/cypress-io/cypress-documentation/pull/4752/commits/9c9f4154e4d078e41217451a906a0de7f1064314